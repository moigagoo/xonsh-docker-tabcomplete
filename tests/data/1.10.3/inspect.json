{
        "arguments": [
                "-f", "--format",
                "--help",
                "-s", "--size",
                "--type"
        ],
        "commands": []
}
