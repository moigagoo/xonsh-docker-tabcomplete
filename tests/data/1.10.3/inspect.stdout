
Usage:	docker inspect [OPTIONS] CONTAINER|IMAGE [CONTAINER|IMAGE...]

Return low-level information on a container or image

  -f, --format       Format the output using the given go template
  --help             Print usage
  -s, --size         Display total file sizes if the type is container
  --type             Return JSON for specified type, (e.g image or container)
